src/test/app/AccountTxPaging_test.cpp

```diff
@@ -16,243 +16,259 @@
-        std::int32_t const page_length = 200;
-        bool const admin = true;
-
-        test::jtx::Env env(*this);
-        Application& app = env.app();
-        auto& txs = txs_;
-
-        auto bound = [&txs, &app](
-            std::uint32_t ledger_index,
-            std::string const& status,
-            Blob const& rawTxn,
-            Blob const& rawMeta)
-        {
-            convertBlobsToTxResult (
-                txs, ledger_index, status, rawTxn, rawMeta, app);
-        };
-
-        accountTxPage(*db_, *idCache_, [](std::uint32_t){}, bound, account_, minLedger,
-            maxLedger, forward, token, limit, admin, page_length);
-
-        return txs_.size();
+        Json::Value jvc;
+        jvc[jss::account] = account.human();
+        jvc[jss::ledger_index_min] = ledger_min;
+        jvc[jss::ledger_index_max] = ledger_max;
+        jvc[jss::forward] = forward;
+        jvc[jss::limit] = limit;
+        if (marker)
+            jvc[jss::marker] = marker;
+
+        return env.rpc("json", "account_tx", to_string(jvc))[jss::result];
     }
 
     void
     testAccountTxPaging ()
     {
-        using namespace std::placeholders;
-
-        bool const forward = true;
-
-        std::int32_t min_ledger;
-        std::int32_t max_ledger;
-        Json::Value token;
-        int limit;
-
-        // the supplied account-tx-transactions.db contains contains
-        // transactions with the following ledger/sequence pairs.
-        //  3|5
-        //  4|4
-        //  4|10
-        //  5|4
-        //  5|7
-        //  6|1
-        //  6|5
-        //  6|6
-        //  6|7
-        //  6|8
-        //  6|9
-        //  6|10
-        //  6|11
-
-        min_ledger = 2;
-        max_ledger = 5;
-
-        {
-            limit = 2;
+        testcase("Paging for Single Account");
+        using namespace test::jtx;
 
-            BEAST_EXPECT(next(limit, forward, token, min_ledger, max_ledger) == 2);
-            checkTransaction (txs_[0], 3, 5);
-            checkTransaction (txs_[1], 4, 4);
-            checkToken (token, 4, 10);
+        Env env(*this);
+        Account A1 {"A1"};
+        Account A2 {"A2"};
+        Account A3 {"A3"};
 
-            BEAST_EXPECT(next(limit, forward, token, min_ledger, max_ledger) == 2);
-            checkTransaction (txs_[0], 4, 10);
-            checkTransaction (txs_[1], 5, 4);
-            checkToken (token, 5, 7);
+        env.fund(XRP(10000), A1, A2, A3);
+        env.close();
 
-            BEAST_EXPECT(next(limit, forward, token, min_ledger, max_ledger) == 1);
-            checkTransaction (txs_[0], 5, 7);
-
-            BEAST_EXPECT(! token["ledger"]);
-            BEAST_EXPECT(! token["seq"]);
-        }
-
-        token = Json::nullValue;
-
-        min_ledger = 3;
-        max_ledger = 9;
+        env.trust(A3["USD"](1000), A1);
+        env.trust(A2["USD"](1000), A1);
+        env.trust(A3["USD"](1000), A2);
+        env.close();
 
+        int count = 0;
+        while (count < 5)
```

Comment:
yeah, I can change that. `do 5.times` is what I really want here, and a for loop is fine.